pdata/ptrace/traces.go

```diff
@@ -46,7 +53,7 @@ func (ms Traces) MoveTo(dest Traces) {
 
 // CopyTo copies the Traces instance overriding the destination.
 func (ms Traces) CopyTo(dest Traces) {
-	ms.ResourceSpans().CopyTo(dest.ResourceSpans())
+	ms.ResourceSpans().CopyTo(dest.MutableResourceSpans())
 }
```

Comment:
Clone.

Reply:
After wrapping the `internal.Traces` into another struct, this func should work as expected. Do you think we still need to change it to `Clone`?